packages/paste-website/src/component-examples/ElevationFoundationExamples.tsx

```diff
@@ -0,0 +1,315 @@
+            borderStyle="solid"
+            borderColor="transparent"
+          >
+            <Box
+              height="100px"
+              borderRadius="borderRadius30"
+              borderLeft="none"
+              borderTopLeftRadius="borderRadius0"
+              borderBottomLeftRadius="borderRadius0"
+              boxShadow="shadowElevation10"
+            />
+            <Text as="p" color="colorTextWeak" marginTop="space50" paddingLeft="space80">
+              Shadow (accent)
+            </Text>
+            <Text as="p" marginTop="space30" paddingLeft="space80">
+              $shadow-elevation-10
+            </Text>
+          </Box>
+        </Theme.Provider>
+      </Box>
+    </Box>
+  );
+};
+
+render(
+  <ShadowTokensLevel2 />
+)
+`.trim();
+
+export const ShadowTokensLevel3Example = `
+const ShadowTokensLevel3 = () => {
+  return (
+    <Box display="flex" fontSize="fontSize30" fontWeight="fontWeightSemibold">
+      <Box
+        width="50%"
+        height="300px"
+        paddingTop="space130"
+        paddingLeft="space80"
+        borderWidth="borderWidth10"
+        borderStyle="solid"
+        borderColor="colorBorderWeaker"
+        zIndex="zIndex10"
+      >
+        <Box
+          height="100px"
+          borderRadius="borderRadius30"
+          borderRight="none"
+          borderTopRightRadius="borderRadius0"
+          borderBottomRightRadius="borderRadius0"
+          boxShadow="shadowElevation20"
+        />
+        <Text as="p" color="colorTextWeak" marginTop="space50">
+          Shadow (accent)
+        </Text>
+        <Text as="p" marginTop="space30">
+          $shadow-elevation-20
+        </Text>
+      </Box>
+      <Box width="50%">
+        <Theme.Provider theme="twilio-dark">
+          <Box
+            backgroundColor="colorBackgroundBody"
+            height="300px"
+            paddingTop="space130"
+            paddingRight="space80"
+            borderWidth="borderWidth10"
+            borderLeftWidth="borderWidth0"
+            borderStyle="solid"
+            borderColor="transparent"
+          >
+            <Box
+              height="100px"
+              borderRadius="borderRadius30"
+              borderLeft="none"
+              borderTopLeftRadius="borderRadius0"
+              borderBottomLeftRadius="borderRadius0"
+              boxShadow="shadowElevation20"
+            />
+            <Text as="p" color="colorTextWeak" marginTop="space50" paddingLeft="space80">
+              Shadow (accent)
+            </Text>
+            <Text as="p" marginTop="space30" paddingLeft="space80">
+              $shadow-elevation-20
+            </Text>
+          </Box>
+        </Theme.Provider>
+      </Box>
+    </Box>
+  );
+};
+
+render(
+  <ShadowTokensLevel3 />
+)
+`.trim();
+
+export const ApplyingLevel1ImgCaption = (): JSX.Element => (
+  <ImageCaption>
+    For example: The Side Panel pushes content at Level 0 but is categorized as Level 1 because it contains a task that
```

Comment:
```suggestion
    For example: The Side Panel pushes content at Level 0 but is categorized as Level 1, and uses $shadow-elevation-05 because it contains a task that
```